### DISCUSSION TITLE
airflow can't load log in interface

### DISCUSSION THREAD

**Discussion by guyttfgl** [apache/airflow#25014] [Q&A]
![interface](https://user-images.githubusercontent.com/38412208/178646241-f3e46826-771e-4152-903a-5f9505be71fc.png)
![2](https://user-images.githubusercontent.com/38412208/178646249-46c0b713-17bd-40fb-9ee4-1c44419a475e.png)
i  have checked the auth of airflow logs directory  , but it also can't load logs in interfacer  , it always loading！and  i can't find any reason

**Comment by potiuk** [comment]
You need to look for errors in webserver.log
